src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  if (sourceString) {
+    return sourceString
+      .split(';')
+      .map((row) => row.split(':').map((param) => param.trim()))
+      .filter((row) => row.length > 1)
+      .map((e) => ({ [e[0]]: e[1] }))
+      .reduce((prew = {}, e) => ({ ...prew, ...e }));
+  } else {
+    return {};
+  }
 }
```

Comment:
The function can be made more concise by chaining the methods without using intermediate variables. Also, ensure that all chained methods have the same indentation level for better readability.